src/commands/python.rs

```diff
@@ -5,13 +5,15 @@ use crate::{
 use clap::Args;
 use indicatif::ProgressBar;
 use serde::Serialize;
-use std::{ffi::OsString, time::Duration};
+use std::time::Duration;
 
 #[derive(Args, Debug, Serialize)]
 #[command(author, version, about)]
 pub struct Python {
+    #[arg(short = 'm', help = "run library module as a script")]
+    pub module: Option<String>,
     #[arg(last = true)]
-    pub slop: Vec<OsString>,
+    pub python_args: Vec<String>,
```

Comment:
Hmm why do it this way and not `aqora python -- -m module`? I feel like this may be a little confusing because both would work or? Also why get rid of OsString?

Reply:
> Hmm why do it this way and not `aqora python -- -m module`?

That way it's documented. I don't mean that we should document every python flags, but this one seems important enough to be shown.

> I feel like this may be a little confusing because both would work or?

Yep, ultimately the last appearance will have precedence I think.

> Also why get rid of OsString?

I think I got lazy there, because I need to join these arguments with a space. I'll try to revert that back.